app/services/paypal_service/store/paypal_payment.rb

```diff
@@ -161,15 +161,22 @@ def create_payment_update(update, current_state)
     payment_update = {}
 
     new_status = transform_status(update[:payment_status]) if update[:payment_status]
+
     new_pending_reason = transform_pending_reason(update[:pending_reason])
     new_state = to_state(new_status, new_pending_reason) if new_status
 
+    commission_new_status = transform_status(update[:commission_status]) if update[:commission_status]
+    commission_new_pending_reason = transform_pending_reason(update[:commission_pending_reason])
+
     if(new_state && valid_transition?(current_state, new_state))
       payment_update[:payment_status] = new_status
       payment_update[:pending_reason] = new_pending_reason
     end
 
-    payment_update[:commission_status] = transform_status(update[:commission_status]) if update[:commission_status]
+    if (commission_new_status)
```

Comment:
No it wouldn't be ignored. It's only skipped if the incoming message doesn't contain any data about commission_status and commission_status is nil. This happens most of time because normal payment updates don't include commission payment information.